end-to-end-rest/docker-compose.yml

```diff
@@ -9,13 +9,13 @@ services:
       - "9010:8000"
 
   skema-tr:
-    image: enoriega87/skema-webapp
+    image: lumai/askem-skema-text-reading
     ports:
-      - "9005:9000"
+      - "9000:9000"
     environment:
       secret: 8c2fb5a58ae8d9ec6f7065a25c35aac2
       SKEMA_HOSTNAME: skema.clulab.org
-      _JAVA_OPTIONS: -Xmx20g -Xms20g -Dfile.encoding=UTF-8
+      _JAVA_OPTIONS: -Xmx10g -Xms10g -Dfile.encoding=UTF-8
```

Comment:
Is this the minimum RAM needed, @enoriega ?

Reply:
Yes.  Hopefully we will be able to bring it down soon